libs/model-ad/not-found/jest.config.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+export default {
+  displayName: 'web-not-found',
+  preset: '../../../jest.preset.js',
+  setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
+  globals: {},
+  coverageDirectory: '../../../coverage/libs/openchallenges/not-found',
```

Comment:
Should this be updated too?

```suggestion
  coverageDirectory: '../../../coverage/libs/model-ad/not-found',
```

Reply:
Fixed